ED/protocol.html

```diff
@@ -1067,7 +1067,9 @@ <h4 property="schema:name">Modifying Resources Using N3 Patches</h4>
                       <li id="server-patch-n3-single" rel="spec:requirement" resource="#server-patch-n3-single"><span property="spec:statement">The patch document <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain exactly one patch resource, identified by one or more of the triple patterns described above, which all share the same <code>?patch</code> subject.</span></li>
                       <li id="server-patch-n3-simple-type" rel="spec:requirement" resource="#server-patch-n3-type">A patch resource <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> contain a triple <code><span property="spec:statement">?patch rdf:type solid:InsertDeletePatch</span></code>.</li>
                       <li id="server-patch-n3-variables" rel="spec:requirement" resource="#server-patch-n3-variables"><span property="spec:statement">The <code>?insertions</code> and <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain variables that do not occur in the <code>?conditions</code> formula.</span></li>
-                      <li id="server-patch-n3-blank-nodes" rel="spec:requirement" resource="#server-patch-n3-blank-nodes"><span property="spec:statement">The <code>?insertions</code> and <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain blank nodes.</span></li>
+                      <li id="server-patch-n3-blank-nodes" rel="spec:requirement" resource="#server-patch-n3-blank-nodes"><span property="spec:statement">The <code>?deletions</code> formulae <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> contain blank nodes.</span></li>
+                      <li id="server-patch-n3-blank-nodes-syntax" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-syntax"><span property="spec:statement">The same blank node identifier <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> appear in more than one scope.</span></li>
+                      <li id="server-patch-n3-blank-nodes-multiplicity" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-multiplicity"><span property="spec:statement">All blank nodes explicitly added in the <code>?insert</code> statement are to be treated as distinct from existing blank nodes in the document.
```

Comment:
```suggestion
                      <li id="server-patch-n3-blank-nodes-multiplicity" rel="spec:requirement" resource="#server-patch-n3-blank-nodes-multiplicity"><span property="spec:statement">All blank nodes in the <code>?insertions</code> statement are to be treated as distinct from existing blank nodes in the target document.
```

I think you meant `?insertions`.

The "document" at the end can be more clear if what you've intended is "patch document" or "target document" (or target resource). Seems more like the latter.

That said, is this actually repeating https://solidproject.org/ED/protocol#server-patch-n3-semantics :
>The triples resulting from ?insertions are to be added to the RDF dataset, with each blank node from ?insertions resulting in a newly created blank node.

Reply:
Good point, it is repeating - let's remove it

```suggestion
```